### PR TITLE
EVA-3567 - Use a static nextflow pipeline 

### PR DESCRIPTION
**Pull request by tcezard**
This PR change the way the nextflow process is handled by having a static version of the nextflow pipeline and running it with parameters.
The nextflow pipeline is still fully linear with with dependency managed by dummy flags.
It also modified the config file to be closer to the one used in eva-submission.

### CODE REVIEW

File: eva-accession-release-automation/run_release_in_embassy/run_release_for_species.py [hunk 1]
```diff
@@ -13,185 +13,125 @@
...
     argparse = ArgumentParser()
-    argparse.add_argument("--common-release-properties-file", help="ex: /path/to/release/properties.yml", required=True)
-    argparse.add_argument("--taxonomy-id", help="ex: 9913", required=True)
-    argparse.add_argument("--assembly-accessions", nargs='+', help="ex: GCA_000003055.3")
+    argparse.add_argument("--taxonomy_id", help="ex: 9913", required=True)
+    argparse.add_argument("--assembly_accessions", nargs='+', help="ex: GCA_000003055.3")
+    argparse.add_argument("--release_version", required=True)
+    argparse.add_argument("--resume", default=False, required=False,
+                          help="Resume the nextflow pipeline for the specified taxonomy and assembly")
+    argparse.add_argument("--release_config_file",
```
Comment (apriltuesday) [inline eva-accession-release-automation/run_release_in_embassy/run_release_for_species.py:124]: Do you think this is needed? I think elsewhere, if the config can be set by an environment variable we don't have command-line argument for it as well, but maybe you have a use case in mind.

File: eva-accession-release-automation/run_release_in_embassy/run_release_for_species.py [hunk 2]
```diff
@@ -13,185 +13,125 @@
...
+    config_param = os.path.join(release_dir, f'nextflow_params_{taxonomy_id}_{assembly_accession}.yaml')
+    os.makedirs(dump_dir, exist_ok=True)
+    yaml_data = {
+        'assembly': assembly_accession,
+        'dump_dir': dump_dir,
+        'executable': cfg['executable'],
+        'jar': cfg['jar'],
+        'log_file': get_release_log_file_name(taxonomy_id, assembly_accession),
+        'maven': cfg['maven'],
+        'python_script': cfg.query('python', 'interpreter'),
```
Comment (apriltuesday) [inline eva-accession-release-automation/run_release_in_embassy/run_release_for_species.py:43]: I think this is accessed from the `executable` section... `params.executable.python.interpreter` in the nextflow

File: eva-accession-release-automation/run_release_in_embassy/run_release_for_species.py (hunk 1 above)
Comment (tcezard) [reply eva-accession-release-automation/run_release_in_embassy/run_release_for_species.py:124]: I was mostly mimicking the previous behaviour. You're right it is not required

File: eva-accession-release-automation/run_release_in_embassy/run_release_for_species.py (hunk 2 above)
Comment (tcezard) [reply eva-accession-release-automation/run_release_in_embassy/run_release_for_species.py:43]: Removed

File: eva-accession-release-automation/run_release_in_embassy/run_release_for_assembly.nf [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+process initiate_release_status_for_assembly {
+
+    label 'short_time', 'med_mem'
+
+    input:
+        val flag
+    output:
+        val true, emit: flag
+    script:
```
[the remark this answers is not in the document]
Comment (tcezard) [reply eva-accession-release-automation/run_release_in_embassy/run_release_for_assembly.nf:20]: Changed it.

**Review by apriltuesday — APPROVED** [review]

**Review by nitin-ebi — APPROVED** [review]

**Pull request merged**